styles.css

```diff
@@ -375,17 +373,58 @@ body {
 
 h1, h2, h3, h4, h5, h6 {
   font-family: "Fjalla One", sans-serif;
+  font-weight: normal;
+}
+
+h1 {
+  font-size: 5rem;
+  text-transform: uppercase;
+  line-height: 130%;
+  letter-spacing: -0.01em;
+}
+
+h2 {
+  font-size: 3rem;
+  margin-bottom: 1rem;
+}
+
+h3 {
+  font-size: 4rem;
+}
+
+h4 {
+  font-size: 3.5rem;
+}
+
+h5 {
+  font-size: 3rem;
+}
+
+h6 {
+  font-size: 2.5rem;
+}
+
+@media screen and (min-width: 700px) {
+  h1 {
+    font-size: 8rem;
+  }
+
+  h2 {
+    font-size: 5rem;
+  }
 }
 
 p {
   font-weight: 300;
+  letter-spacing: -0.06em;
+  font-size: 1.8rem;
+  line-height: 150%;
```

Comment:
Line heights are best expressed as a unit-less value, due to children being able to calculate the ratio based on their own font size. See https://css-tricks.com/almanac/properties/l/line-height/#unitless-line-heights